entity-management-web/src/main/java/eu/europeana/entitymanagement/web/xml/model/XmlMultilingualString.java

```diff
@@ -25,7 +25,12 @@ public XmlMultilingualString() {
 
     public XmlMultilingualString(String value, String language) {
         this.value = value;
-        this.language = language;
+        if(language != null) {
+            this.language = language;
+        } else {
+            //fix for #EA-2325, missing language attributions changed to ""
+	    this.language = "";  
```

Comment:
This has no effect, as JAXB uses the zero-arg constructor by default during deserialization.

Putting this logic in the setter wouldn't work either – as the missing `xml:lang` attribute means that won't be invoked.

So the best place to perform the null check is in `getLanguage()`.

Reply:
I added the default value as "", and I added the not null check to the getLanguage() as well